rest-api-spec/src/main/resources/rest-api-spec/test/index/110_constant_keyword.yml

```diff
@@ -1,17 +1,18 @@
 ---
 # The test setup includes:
-# - Create index with constant_keyword field type
+# - Create two indices with constant_keyword field type
+# - Index documents
 # - Check mapping
-# - Index two example documents
-# - Search
-# - Delete Index when connection is teardown
+# - Verify document count
+# - Search rangeQuery and regexpQuery
+# - Delete indices when connection is teardown
 
 "Mappings and Supported queries":
   - skip:
```

Comment:
@kkewwei I believe it is too late for 2.16, we would need 2.17 check for new test cases

Reply:
Thanks for reminding.

Reply:
Hm ... I think there is confusion here:
 - `constant_keyword` and  `110_constant_keyword.yml` consequently went into 2.16.0 (this check is fine)
 - this bugfix (and consequently new tests) will go to 2.17.0

If we run these new tests without this fix in place (2.16.0), they will fail (or at least should) so for those we need to have:

```
- skip:
   version: " - 2.16.99"
      reason: "fixed in 2.17.0"
```

For existing tests the `skip` condition does not need to be changed.

Reply:
@reta how about creating a new file `120_constant_keyword_query.yml`, or temporarily delete the unit test and add later?

Reply:
> @reta, can you help see it? I have deviced into two parts, but it always fails. It has nothing to do with the code itself.

@kkewwei could you please rebase? the fix for this build issue just went in a few minutes ago, thank you